### PR TITLE
Add survey signup to dashboard

### PR DESCRIPTION
**Pull request by lesleyjanenorton**

### CODE REVIEW

File: public/js/monitor.js [hunk 1]
```diff
@@ -410,6 +438,34 @@ function addBentoObserver(){
...
+    }
+  };
+  const promoHasAlreadyBeenDismissed = (researchPromo && sessionStorage.getItem("fxMonitorResearchPromo") === "dismissed");
+  const sessionStorageDoesNotExist = (researchPromo && !sessionStorage);
+  const acceptedLanguages = navigator.languages;
+  const acceptedLocalesIncludesEnglish =
+    (acceptedLanguages.includes("en-US") || acceptedLanguages.includes("en"));
+
+  if (promoHasAlreadyBeenDismissed || sessionStorageDoesNotExist || !acceptedLocalesIncludesEnglish) {
+    researchPromo.style.display = "none";
```
Comment (groovecoder) [inline public/js/monitor.js:454]: Could we save a lot of `researchPromoIsVisible` logic if we `return;` here?

File: public/js/monitor.js [hunk 2]
```diff
@@ -410,6 +438,34 @@ function addBentoObserver(){
       });
     }
   }
+  const researchPromo = document.querySelector(".research-recruitment-wrapper");
+  const sendRecruitmentPing = (action, interactionStatus) => {
+    if (gaAvailable) {
+      ga("send", "event", "Research Recruitment Promo", action, "Recruitment Promo", {nonInteraction: interactionStatus});
```
Comment (maxxcrawford) [inline public/js/monitor.js:444]: You'll need to send additional options on the click/Engage event (but not the "view" event) 
`{transport: "beacon"}`

Similar to this: https://github.com/mozilla/blurts-server/blob/master/public/js/fxa-analytics.js#L75

File: public/js/monitor.js (hunk 1 above)
Comment (lesleyjanenorton) [reply public/js/monitor.js:454]: We could! Thank you!

**Comment by groovecoder** [comment]
r+

**Comment by maxxcrawford** [comment]
r+

**Pull request merged**